### DISCUSSION TITLE
Scrollable Frame background

### DISCUSSION THREAD

**Discussion by ubaid2751** [TomSchimansky/CustomTkinter#2403] [Q&A]
The frame has the corner radius but at its back the black color is still there? How can I remove it?
![Screenshot 2024-05-09 162618](https://github.com/TomSchimansky/CustomTkinter/assets/68930485/28988bea-c3e4-481c-8296-616743b4a99c)

**Comment by rigvedmaanas** [comment]
Hi @ubaid2751 If you are on windows then you can take a look at this issue [#2214](https://github.com/TomSchimansky/CustomTkinter/discussions/2214). This will allow you to make the background of a widget transparent.

**Comment by ubaid2751** [comment]
hi @rigvedmaanas I have tried this but I don't know why is it not working for Scrollable Frame as it is working for a normal frame.

**Comment by rigvedmaanas** [comment]
Maybe you should try giving the transparency to the inner component.

```python
scrl_frame = CTkScrollableFrame(master=app, bg_color="#000001") 
scrl_frame.place(relx=0.5, rely=0.5, anchor="center")

pywinstyles.set_opacity(scrl_frame._parent_frame, color="#000001")
```
The above code is not tested because I don't have a windows computer.
